redbot/cogs/trivia/data/lists/hockey.yaml

```diff
@@ -0,0 +1,2097 @@
+- Pageau
+Who has the most hat tricks in Senators playoff history?:
+- Daniel Alfredsson & Jean-Gabriel Pageau (2)
+- Daniel Alfredsson
+- Alfredsson
+- Jean-Gabriel Pageau
+- Pageau
+Who has the most penalty minutes in a single season for the Senators?:
+- Mike Peluso, 318 (1992-93)
+- Mike Peluso
+- Peluso
+Who has the most short-handed goals in a single season for the Senators?:
+- Daniel Alfredsson (2007-08) & Jean-Gabriel Pageau (2015-16) (7)
+- Daniel Alfredsson
+- Alfredsson
+- Jean-Gabriel Pageau
+- Pageau
+Who has the most shots in a single season for the Senators?:
+- Alexei Yashin, 337 (1998-99)
+- Alexei Yashin
+- Yashin
+Who is the only player to wear \#77 in Senators history?:
+- Joe Corvo, 2014-15
+- Joe Corvo
+- Corvo
+Which number was the first retired in Senators history and for which player?:
+- \#8 Frank Finnigan
+- 8
+- eight
+How many times have the Ottawa Senators appeared in the playoffs?:
+- 16 times
+- 16
+- sixteen
+How many seasons have the Ottawa Senators played as a franchise?:
+- 27 seasons
+- 27
+- twentyseven
+How many times have the Ottawa Senators been sweeped in the playoffs?:
+- 3 times (1998-99 vs Buffalo, 2000-01 vs Toronto, 2007-08 vs Pittsburgh)
+- 3
+- three
+Who was the last player the Senators drafted from Russia?:
+- Ruslan Bashkirov, 2nd round 2007
+- Ruslan Bashkirov
+- Bashkirov
+Which draft did the Senators exclusively pick Canadian players?:
+- 2017 (Shane Bowers, Alex Formenton, Drake Batherson, Jordan Hollett)
+- 2017
+Which draft did the Senators not own a 1st or a 2nd round pick?:
+- 2010
+Who was the first Ottawa Senators head coach?:
+- Rick Bowness
+- Bowness
+Who was the first Ottawa Senators general manager?:
+- Mel Bridgman
+- Bridgman
+How many head coaches have the Senators had throughout their history?:
+- 13 (Bowness, Allison, Martin, Neilson, Murray, Paddock, Hartsburg, Clouston, MacLean, Cameron, Boucher, Crawford, Smith)
+- 13
+How many general managers have the Senators had throughout their history?:
+- 8 (Bridgman, Sexton, Gauthier, Dudley, Johnston, Muckler, Murray, Dorion)
+- 8
+Who was the first player in Senators history to lead the team in goals through a season?:
+- Sylvain Turgeon, 25
+How many playoff series have the Senators appeared in?:
+- 27
+- twenty seven
+How many playoff series have the Senators won?:
+- 11
+- eleven
+Which eastern conference teams have the Senators NOT faced in the playoffs?:
+- Columbus, Detroit, Carolina, Florida
+- Columbus Blue Jackets
+- Columbus
+- Blue Jackets
+- Detroit Red Wings
+- Detroit
+- Red Wings
+- Carolina Hurricanes
+- Carolina
+- Hurricanes
+- Florida Panthers
+- Florida
+- Panthers
+How many times have the Senators had the best record in their division?:
+- 4 times (1998-99, 2000-01, 2002-03, 2005-06)
+- 4
+- four
+How many times have the Senators finished last place in their division?:
+- 6 times (1992-93, 1993-94, 1994-95, 1995-96, 2010-11, 2018-19)
+- 6
+- six
+What is the Senators longest streak of making the playoffs?:
```

Comment:
Maybe make it clear in the question if it's seasons or years and remove 'seasons' from the responses